rust/tvm-macros/src/external.rs

```diff
@@ -124,15 +151,21 @@ pub fn macro_impl(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
                         let ty: Type = *pat_type.ty.clone();
                         (ident, ty)
                     }
-                    _ => panic!(),
+                    _ => abort! { pat_type,
```

Comment:
Is `abort!` the standard way to blow up a macro?

Reply:
This is proc_macro_error which is crate for doing better errors in proc macros.